issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraProject.java

```diff
@@ -523,10 +506,14 @@ private List<IssueTrackerIssue> queryIssues(String jql) {
 
             if (count < issues.get("total").asInt()) {
                 startAt += issues.get("issues").asArray().size();
```

Comment:
Aren't `count` and `startAt` basically equivalent at this point? Seems a bit weird to have two values for basically the same thing but calculated differently.

Reply:
Yes, I just tried to change the existing code as little as possible to try to keep the patch about refactoring. Since I will be touch this a bit more based on your other comment then I can clean this up as well.